samples/bot-task-module/csharp/Helper/AdaptiveCardHelper.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

Reply:
fixed